packages/web-components/src/components/card-group/__stories__/card-group.stories.react.tsx

```diff
@@ -48,184 +47,229 @@ const phraseArray = [
 ];
 
 const cardsCol = {
-  '3 cards per row (Default)': 'dds-ce-demo-devenv--cards-in-row-3',
   '2 cards per row': 'dds-ce-demo-devenv--cards-in-row-2',
+  '3 cards per row (default)': 'dds-ce-demo-devenv--cards-in-row-3',
   '4 cards per row': 'dds-ce-demo-devenv--cards-in-row-4',
 };
 
-const cardsDiffLengthPhrase = (index, border, tagGroup, defaultSrc, image) => {
+const gridModes = {
+  [`Collapsed (1px)`]: GRID_MODE.COLLAPSED,
+  [`Narrow (16px)`]: GRID_MODE.NARROW,
+  [`Outlined cards (1px border)`]: GRID_MODE.BORDER,
+};
+
+const tagGroupContent = (
+  <DDSTagGroup>
+    <Tag type="cool-gray">Systems w/TPS</Tag>
+    <Tag type="cool-gray">Virtual</Tag>
+  </DDSTagGroup>
+);
+
+const imageContent = <DDSCardCTAImage slot="image" alt="Image alt text" defaultSrc={imgXlg4x3} />;
+
+const cardsDiffLengthPhrase = (index, tagGroup, media, gridMode, cardType) => {
   const defaultCardGroupItem = (
-    <DDSCardGroupItem cta-type="local" href="https://example.com" color-scheme={border ? 'light' : null}>
-      {image ? <DDSImage slot="image" alt="Image Alt text" defaultSrc={defaultSrc || undefined} /> : ''}
+    <DDSCardGroupItem
+      cta-type={cardType === 'Card static' ? '' : 'local'}
+      href={cardType === 'Card static' ? '' : 'https://example.com'}
+      colorScheme={cardType === 'Card static' || gridMode === 'border' ? 'light' : null}>
+      {media ? imageContent : ''}
       <DDSCardEyebrow>Topic</DDSCardEyebrow>
       <DDSCardHeading>{index < 5 ? phraseArray[index] : 'Lorem ipsum dolor sit amet'}</DDSCardHeading>
       <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est.'</p>
-      {tagGroup ? (
-        <DDSTagGroup>
-          <Tag type="cool-gray">Systems w/TPS</Tag>
-          <Tag type="cool-gray">Virtual</Tag>
-        </DDSTagGroup>
-      ) : (
-        ''
-      )}
-      <DDSCardCTAFooter slot="footer"></DDSCardCTAFooter>
+      {tagGroup ? tagGroupContent : ''}
+      {cardType === 'Card static' ? '' : <DDSCardCTAFooter slot="footer"></DDSCardCTAFooter>}
+    </DDSCardGroupItem>
+  );
+
+  const videoCardGroupItem = (
+    <DDSCardGroupItem cta-type="video" href="1_9h94wo6b" color-scheme={gridMode === 'border' ? 'light' : null}>
+      <DDSCardEyebrow>Topic</DDSCardEyebrow>
+      {tagGroup ? tagGroupContent : ''}
+      <DDSCardCTAFooter cta-type="video" href="1_9h94wo6b" slot="footer"></DDSCardCTAFooter>
     </DDSCardGroupItem>
   );
+
   count = count > 3 ? 0 : count + 1;
-  return defaultCardGroupItem;
+  // eslint-disable-next-line no-nested-ternary
+  return media ? (index % 2 === 0 ? defaultCardGroupItem : videoCardGroupItem) : defaultCardGroupItem;
```

Comment:
```suggestion
  return media && index % 2 ? videoCardGroupItem : defaultCardGroupItem;
```

would this be equivalent without nesting ternaries?